### PR TITLE
Upgrades policy/v1beta1 to policy/v1 

### PR DESCRIPTION
**Pull request by saithal-confluent**
Jira: https://confluentinc.atlassian.net/browse/OBSDATA-986

### CODE REVIEW

**Comment by saithal-confluent** [comment]
Note: The deepcopy object is not updated

File: controllers/druid/handler.go [hunk 1]
```diff
@@ -979,18 +979,18 @@ func getVolume(nodeSpec *v1alpha1.DruidNodeSpec, m *v1alpha1.Druid, nodeSpecUniq
 }
 
 func getCommand(nodeSpec *v1alpha1.DruidNodeSpec, m *v1alpha1.Druid) []string {
-  if (m.Spec.StartScript != "" && m.Spec.EntryArg != "") {
```
Comment (hampycapper) [inline controllers/druid/handler.go:982]: This and the next set of changes look like non-semantic changes only. You might want to leave these for a different diff to ensure the policyv1 changes don't conflict with other unrelated changes but that happen to touch the same lines.

File: controllers/druid/handler.go [hunk 2]
```diff
@@ -1360,10 +1360,10 @@ func makeDeloymentListEmptyObj() *appsv1.DeploymentList {
 	}
 }
 
-func makePodDisruptionBudgetListEmptyObj() *v1beta1.PodDisruptionBudgetList {
-	return &v1beta1.PodDisruptionBudgetList{
+func makePodDisruptionBudgetListEmptyObj() *policyv1.PodDisruptionBudgetList {
+	return &policyv1.PodDisruptionBudgetList{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "policy/v1beta1",
```
Comment (hampycapper) [inline controllers/druid/handler.go:1366]: Any chance we can make this a constant somewhere and use that constant here and below?

**Review by hampycapper — COMMENTED** [review]
Holding off on approving for you to check for a simliar fix from upstream.

**Pull request closed without merging**